app/src/main/java/de/othr/plantico/database/PlanticoDB.kt

```diff
@@ -4,16 +4,21 @@ import android.content.Context
 import androidx.room.Database
 import androidx.room.Room
 import androidx.room.RoomDatabase
+import androidx.room.TypeConverters
 import androidx.sqlite.db.SupportSQLiteDatabase
+import de.othr.plantico.database.daos.OwnedPlantDAO
 import de.othr.plantico.database.daos.PlantDAO
-import de.othr.plantico.database.entities.Plant
+import de.othr.plantico.database.entities.*
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.launch
 
-@Database(entities = arrayOf(Plant::class), version = 1, exportSchema = false)
+@Database(entities = arrayOf(Plant::class, OwnedPlant::class), version = 1, exportSchema = false)
```

Comment:
About the problem, you mentioned on Discord. Does it help if you bump up the database version here? This could indicate old app versions, that it has to be updated. But not sure about it.